src/supervisor/watchers/handlers/namespace.ts

```diff
@@ -127,7 +134,7 @@ export async function trackNamespace(namespace: string): Promise<void> {
   informer.on(ERROR, (error) => {
     const code = error.code || '';
     logContext.code = code;
-    if (RETRYABLE_NETWORK_ERRORS.includes(code)) {
+    if (RETRYABLE_NETWORK_ERROR_CODES.includes(code)) {
```

Comment:
Should this one also include `RETRYABLE_NETWORK_ERROR_MESSAGES.includes(message)` too?